### PR TITLE
8282642: vmTestbase/gc/gctests/LoadUnloadGC2/LoadUnloadGC2.java fails intermittently with exit code 1

### PR DESCRIPTION
**Pull request by rsunderbabu**
LoadUnloadGC2 tests the Class unloading during GC. In order to trigger the GC, lots of memory is created at a short interval. Instead of GC getting triggered, the test quickly gets into OOME.
Using WhiteBox.fullGC is a cleaner and deterministic way to trigger GC.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8282642](https://bugs.openjdk.org/browse/JDK-8282642): vmTestbase/gc/gctests/LoadUnloadGC2/LoadUnloadGC2.java fails intermittently with exit code 1

### Reviewers
 * [Leonid Mesnik](https://openjdk.org/census#lmesnik) (@lmesnik - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk pull/9628/head:pull/9628` \
`$ git checkout pull/9628`

Update a local copy of the PR: \
`$ git checkout pull/9628` \
`$ git pull https://git.openjdk.org/jdk pull/9628/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 9628`

View PR using the GUI difftool: \
`$ git pr show -t 9628`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/9628.diff">https://git.openjdk.org/jdk/pull/9628.diff</a>

</details>

### CODE REVIEW

File: test/hotspot/jtreg/vmTestbase/gc/gctests/LoadUnloadGC2/LoadUnloadGC2.java [hunk 1]
```diff
@@ -49,10 +52,8 @@ public void run() {
                 stresser.start(500000);
                 try {
                         while (stresser.iteration()) {
-                                GarbageProducer garbageProducer = new GeneratedClassProducer();
                                 log.info("Iteration: " + stresser.getIteration());
-                                GarbageUtils.eatMemory(stresser, garbageProducer, 0);
-                                garbageProducer = null;
+                                WhiteBox.getWhiteBox().fullGC();
```
Comment (lmesnik) [inline test/hotspot/jtreg/vmTestbase/gc/gctests/LoadUnloadGC2/LoadUnloadGC2.java:56]: We need to create some classes to unload them. Are they created?

Comment (rsunderbabu) [reply test/hotspot/jtreg/vmTestbase/gc/gctests/LoadUnloadGC2/LoadUnloadGC2.java]: I missed that part. I will correct it.

**Review by lmesnik — APPROVED** [review]
Looks good, please verify that it works with ZGC, ParallelGC, SerialGC.

**Comment by rsunderbabu** [comment]
> Looks good, please verify that it works with ZGC, ParallelGC, SerialGC.

@lmesnik I have tested with all the GC options.

**Pull request closed without merging**